app/validators/partner_details/answers_validator.rb

```diff
@@ -3,8 +3,8 @@ class AnswersValidator < BaseAnswerValidator
     include TypeOfMeansAssessment
 
     def applicable?
+      return false if partner_information_not_required?
```

Comment:
👇 I'm not sure why we have the `return false unless record` guard clause here, but again this is a big PR for a bug fix.